sabotage_field/sabotage_dice_result.gd

```diff
@@ -0,0 +1,83 @@
+extends CanvasLayer
+
+@onready var label=$CanvasLayer/Panel/Label
+
+signal finished
+
+var player: Player
+var action: Callable
+
+
+func _on_button_pressed():
+	await action.call()
+	finished.emit()
+
+
+func display(player: Player, dice_value):
+	self.player = player
+	visible = true
+
+	var duel_50 = {
+		"text": "Duell Einsatz: 50 Riesen",
+		"action": func():
+			var target = await get_target()
+			var result = await run_duel(target)
+			# match case over result
+	}
+
+
+	if (dice_value==7 || dice_value==8):
+		result="Big Baller Duell Einsatz: halbes Vermögen"
+		action = duel_big
+	if (dice_value==9):
+		result="Duell um Zug stehlen"
+		action = steal_turn
+	if (dice_value==10):
+		result="Erhalt Sabotage Freikarte"
+		action = sabotage_free_card
+	if (dice_value==11):
+		result="Eigentor Verlust: 50 Riesen"
+		action = lose_50
+	if (dice_value==12):
+		result="Duell Würfelgeld \nWert:gewürfelte Anzahl Riesen*10"
+		action = duel_roll
```

Comment:
Mapping the values should also be done with a Dictionary but at least use `elif` for consecutive statements.